rebar.lock

```diff
@@ -1,7 +1,7 @@
 {"1.1.0",
 [{<<"aebytecode">>,
   {git,"https://github.com/aeternity/aebytecode.git",
-      {ref,"04571f757a21916d9e8cbbd51f460445e7f76ccb"}},
+      {ref,"34ae94e3e74fbd2ff36f7e5d601b9fd8318a9832"}},
```

Comment:
I see https://github.com/aeternity/aebytecode/compare/04571f757a21916d9e8cbbd51f460445e7f76ccb...34ae94e3e74fbd2ff36f7e5d601b9fd8318a9832
Only one small fate change, plus aeserialization brought in line with this repo.
Ok.